extensions/fairifier-rdf-extension/scripts/rdf-schema-new-prefix-widget.js

```diff
@@ -81,7 +84,7 @@ NewPrefixWidget.prototype.show = function(msg,def_prefix,onDone){
             });
     	}else{
             dismissBusy = DialogSystem.showBusy('Trying to import vocabulary from ' + uri);
-            $.post("command/rdf-extension/add-prefix",{name:name,uri:uri,"fetch-url":uri,project: theProject.id,fetch:fetchOption},function(data){
```

Comment:
Why is the fetchoption removed? (relates to the comment on `AddPrefixCommand` as well)